src/randgraphs.jl

```diff
@@ -96,11 +96,10 @@ end
 
 function _suitable(edges::Set{Edge}, potential_edges::Dict{Int, Int})
     isempty(potential_edges) && return true
-    for (s1, s2) in combinations(collect(keys(potential_edges)), 2)
-        if (s1 > s2)
-            s1, s2 = s2, s1
-        end
-        ∉(Edge(s1, s2), edges) && return true
+    list = keys(potential_edges)
+    for s1 in list, s2 in list
```

Comment:
Let's keep it as is. It's straightforward and probably not worth the time to optimize at this point.